src/components/StatusBadge/_StatusBadge.scss

```diff
@@ -0,0 +1,16 @@
+.status-badge {
+    color: var(--ui-black);
+    display: block;
+    font-style: italic;
+    margin-bottom: calc(var(--space-m) / 2);
+    padding: calc(var(--space-s) / 2);
```

Comment:
Is this padding intended?  With other components, we've had it stick to its container - we should probably be consistent

Reply:
I'm not sure what you mean by padding sticking to its container with other components... are you maybe referring to the global decorator I added? That has caused every story to have some space from the edges of the browser: https://github.com/NYPL/nypl-design-system/blob/development/.storybook-react/config.js#L12

It was a good comment though, I don't think I should have added that marginBottom (that's a holdout from the CSS I copied over from PatternLab), and it looks like I was missing a border-radius when I looked at the designs more carefully 😅

Reply:
guhhhh ignore me.  The components used to stick to their container but now there is a global decorator which my brain forgot about, dumb brain.  So the padding threw me and then I thought this was the padding ...   Glad that you did find real things to fix though haha